ui-spacetimechart/src/components/PathLayer.tsx

```diff
@@ -237,18 +245,20 @@ export const PathLayer: FC<PathLayerProps> = ({
       ctx.save();
       ctx.translate(position.x, position.y);
       ctx.rotate(angle);
-      ctx.font = `${FONT_SIZE}px IBM Plex Mono`;
+      ctx.font = font;
       ctx.textAlign = 'start';
 
       const dx = 5;
       const dy = angle >= 0 ? -5 : 15;
       const padding = 2;
       const measure = ctx.measureText(label);
       const w = measure.width + 2 * padding;
-      const h = FONT_SIZE + 2 * padding;
-      ctx.fillStyle = WHITE_75;
+      const h = measure.actualBoundingBoxAscent + measure.actualBoundingBoxDescent + 2 * padding;
```

Comment:
Can you add a intermediate const here ? I'm not used to see `measure.actualBoundingBoxAscent` and `measure.actualBoundingBoxDescent`
```suggestion
      const fontSize = measure.actualBoundingBoxAscent + measure.actualBoundingBoxDescent;
      const h = fontSize + 2 * padding;
```

Reply:
(I agree, but it won't be named `fontSize`, since it does not compute a font size, but the actual text bounding box height)